writing-your-first-es2015-javascript-module.md

```diff
@@ -0,0 +1,140 @@
+<!--
+headline: Writing Your First ES2015 JavaScript Module
+description: dead simple guide to writing your first ES2015 module using Babel.
+author: Ben Coe
+datePublished: 2015-11-29
+twitter: benjamincoe
+github: bcoe
+-->
+
+> ES2015 introduces a many slick language features (arrow functions, classes, template strings), but it
+  can be overwhelming figure out where to begin with it!
```

Comment:
[grammar] Some grammar mistakes:
- "introduces a many" -> "introduces many"
- "can be overwhelming figure out" -> "can be overwhelming to figure out"